simple_ddl_parser/dialects/sql.py

```diff
@@ -960,8 +967,6 @@ def set_constraint(
             target_dict["constraints"] = {}
         if not target_dict["constraints"].get(_type):
             target_dict["constraints"][_type] = []
-        if "name" in constraint:
```

Comment:
i don't know why these were here, but they were breaking my references.  Removing them did not break any other tests, so i think that maybe they were unnecessary?

Reply:
possible, it is redunant code, looks like I changed some logic, but forgot to remove this